test-data/ert/snake_oil/forward_models/snake_oil_simulator.py

```diff
@@ -19,135 +20,151 @@ def readParameters(filename):
     return params
 
 
-def runSimulator(simulator, history_simulator, time_step_count) -> Summary:
-    summary = Summary.writer("SNAKE_OIL_FIELD", datetime(2010, 1, 1), 10, 10, 10)
-
-    summary.add_variable("FOPT", unit="SM3")
-    summary.add_variable("FOPR", unit="SM3/DAY")
-    summary.add_variable("FGPT", unit="SM3")
-    summary.add_variable("FGPR", unit="SM3/DAY")
-    summary.add_variable("FWPT", unit="SM3")
-    summary.add_variable("FWPR", unit="SM3/DAY")
-    summary.add_variable("FGOR", unit="SM3/SM3")
-    summary.add_variable("FWCT", unit="SM3/SM3")
-
-    summary.add_variable("FOIP", unit="SM3")
-    summary.add_variable("FGIP", unit="SM3")
-    summary.add_variable("FWIP", unit="SM3")
-
-    summary.add_variable("FOPTH", unit="SM3")
-    summary.add_variable("FOPRH", unit="SM3/DAY")
-    summary.add_variable("FGPTH", unit="SM3")
-    summary.add_variable("FGPRH", unit="SM3/DAY")
-    summary.add_variable("FWPTH", unit="SM3")
-    summary.add_variable("FWPRH", unit="SM3/DAY")
-    summary.add_variable("FGORH", unit="SM3/SM3")
-    summary.add_variable("FWCTH", unit="SM3/SM3")
-
-    summary.add_variable("FOIPH", unit="SM3")
-    summary.add_variable("FGIPH", unit="SM3")
-    summary.add_variable("FWIPH", unit="SM3")
-
-    summary.add_variable("WOPR", wgname="OP1", unit="SM3/DAY")
-    summary.add_variable("WOPR", wgname="OP2", unit="SM3/DAY")
-    summary.add_variable("WWPR", wgname="OP1", unit="SM3/DAY")
-    summary.add_variable("WWPR", wgname="OP2", unit="SM3/DAY")
-    summary.add_variable("WGPR", wgname="OP1", unit="SM3/DAY")
-    summary.add_variable("WGPR", wgname="OP2", unit="SM3/DAY")
-    summary.add_variable("WGOR", wgname="OP1", unit="SM3/SM3")
-    summary.add_variable("WGOR", wgname="OP2", unit="SM3/SM3")
-    summary.add_variable("WWCT", wgname="OP1", unit="SM3/SM3")
-    summary.add_variable("WWCT", wgname="OP2", unit="SM3/SM3")
-
-    summary.add_variable("WOPRH", wgname="OP1", unit="SM3/DAY")
-    summary.add_variable("WOPRH", wgname="OP2", unit="SM3/DAY")
-    summary.add_variable("WWPRH", wgname="OP1", unit="SM3/DAY")
-    summary.add_variable("WWPRH", wgname="OP2", unit="SM3/DAY")
-    summary.add_variable("WGPRH", wgname="OP1", unit="SM3/DAY")
-    summary.add_variable("WGPRH", wgname="OP2", unit="SM3/DAY")
-    summary.add_variable("WGORH", wgname="OP1", unit="SM3/SM3")
-    summary.add_variable("WGORH", wgname="OP2", unit="SM3/SM3")
-    summary.add_variable("WWCTH", wgname="OP1", unit="SM3/SM3")
-    summary.add_variable("WWCTH", wgname="OP2", unit="SM3/SM3")
-
-    summary.add_variable("BPR", num=globalIndex(5, 5, 5), unit="BARSA")
-    summary.add_variable("BPR", num=globalIndex(1, 3, 8), unit="BARSA")
+def write_summary_spec(file, keywords, names, units, nums):
+    content = [
```

Comment:
Is is possible to leave a documentation link/hint for how to maintain this section?

Reply:
Hmm, possibly. This function is already copied around a few places.

Reply:
We discussed it a bit and it would probably be a good idea to split this out into its own library.